plugins/htpasswd/CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,46 @@
+# Contributor Covenant Code of Conduct
```

Comment:
I think that in subprojects we do not need a CODE_OF_CONDUCT because it is in main project